packages/forma-36-react-components/src/components/Flex/Flex.tsx

```diff
@@ -0,0 +1,221 @@
+import React from 'react';
+import * as CSS from 'csstype';
+import tokens from '@contentful/forma-36-tokens';
+import cn from 'classnames';
+
+import styles from './Flex.css';
+
+export type SpacingTypes = 'none' | 'spacing2xs' | 'spacingXs' | 'spacingS' | 'spacingM' | 'spacingL' | 'spacingXl' | 'spacing2Xl' | 'spacing3Xl' | 'spacing4Xl';
+
+export type FlexContainerType =
+  | 'a'
+  | 'article'
+  | 'aside'
+  | 'div'
+  | 'footer'
+  | 'form'
+  | 'h1'
+  | 'h2'
+  | 'h3'
+  | 'h4'
+  | 'h5'
+  | 'h6'
+  | 'header'
+  | 'input'
+  | 'li'
+  | 'main'
+  | 'nav'
+  | 'ol'
+  | 'p'
+  | 'pre'
+  | 'section'
+  | 'span'
+  | 'textarea'
+  | 'ul';
+
+export type justifyContentValues =
+| 'center'    
+| 'start'     
+| 'end'       
+| 'flex-start'
+| 'flex-end'  
+| 'left'      
+| 'right'
+| 'normal'
+| 'space-between'                                
+| 'space-around'                                   
+| 'space-evenly'                                 
+| 'stretch'                                     
+| 'safe center'
+| 'unsafe center'
+| 'inherit'
+| 'initial'
+| 'unset'
+
+export interface FlexProps {
+  /**
+   * Class names to be appended to the className prop of the component */
+  className?: string;
+  /**
+   * Child nodes to be rendered in the component */
+  children?: React.ReactNode;
+  /**
+   * An ID used for testing purposes applied as a data attribute (data-test-id) */
+  testId?: string;
+  /**
+   * Html tag used as container
+   */
+  htmlTag?: FlexContainerType,
```

Comment:
I can't find types for a full list of all html tags. maybe setting this to a string is better?

Reply:
@mshaaban0 I declared them I bit ⬆️